README.md

```diff
@@ -15,6 +15,8 @@ Handle:
 - list stores (`/api/item/`)
 - get a store (`/api/item/:id`)
 - set favorite (`/api/item/:id/setFavorite`)
+- get active orders (`order/vX/active`)
```

Comment:
should not we add `/api/` like the others endpoint ?
 ```suggestion
- get active orders (`/api/order/vX/active`)
```

Reply:
changed.